.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+name: Main Workflow
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  main:
+    name: Main Job
+    runs-on: macos-12
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache Mint packages
+        id: mint-cache-v2
+        uses: actions/cache@v3
+        with:
+          path: ~/.mint
+          key: ${{ runner.os }}-mint-${{ hashFiles('Mintfile') }}
+          restore-keys: ${{ runner.os }}-mint-
+
+      - name: Install Mint
+        run: brew install mint
+
+      - name: Install Mint Packages
+        if: steps.mint-cache-v2.outputs.cache-hit != 'true'
+        run: mint bootstrap
+
+      - name: Run Swiftformat linter
+        run: mint run swiftformat . --lint
+
+      - name: Run Swiftlint
+        run: mint run swiftlint lint --reporter github-actions-logging
```

Comment:
These are basic `bash` commands. The trick here is the `--reporter github-actions-logging` which tells swiftlint to report in a GH Actions format.

This basically means that the format changes from 
```
/Users/andraskadar/Development/furry/furry-fiesta-ios/Log/Tests macOS/Tests_macOS.swift:10:7: error: Type Name Violation: Type name should only contain alphanumeric characters: 'Tests_macOS' (type_name)
```
to
```
::error file=Log/Tests macOS/Tests_macOS.swift,line=10,col=7::Type name should only contain alphanumeric characters: 'Tests_macOS' (type_name)
```

and you can see in the changed files list how GH will automatically parse these errors and warnings to present in-line with the code.

<img width="1402" alt="Screen Shot 2022-05-02 at 1 08 04 AM" src="https://user-images.githubusercontent.com/1413987/166168319-c91ed9f1-d380-407a-aa97-a66abdc4309b.png">